AirBench/Scripts/map.js

```diff
@@ -104,35 +103,118 @@ map.on('click', clickHandler);
 document.getElementById("minInput").addEventListener("keyup", minFunction)
 document.getElementById("maxInput").addEventListener("keyup", maxFunction)
 
+let filteredbench = [];
+filteredbench = jsonResponse.benchList;
+DisplayBenches(filteredbench)
+let minvalue = undefined;
+let maxvalue = undefined;
 function minFunction (event) {
-    let input = document.getElementById('minInput');
-    let list = document.getElementsByClassName('inner-list');
-    console.log(list);
-    for (let item of list){
-        console.log(item);
-        //let num = item.getElementById('Number');
-        //console.log (num);
+    minvalue = document.getElementById("minInput").value
+    console.log('min value is' + minvalue)
+    console.log('max value is'+ maxvalue)
+    if (!(minvalue==0 || minvalue==undefined)){
+        if (maxvalue==undefined || maxvalue==0){
+            filteredbench = [];
+            for(let bench of jsonResponse.benchList){
+                if (bench['NumberOfSeats']>=minvalue){
+                    filteredbench.push(bench);
+                    console.log(bench['NumberOfSeats'])
+                }
+            }
```

Comment:
Check out the Array `filter` method: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter